app/javascript/src/components/Dashboard/Notes/Menu.jsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
+
+import { Plus, Search, Settings } from "neetoicons";
+import { Typography } from "neetoui";
+import { MenuBar } from "neetoui/layouts";
+
+const Menu = ({ showMenu }) => {
+  const [isSearchCollapsed, setIsSearchCollapsed] = useState(true);
+
+  return (
+    <div className="flex">
+      <MenuBar showMenu={showMenu} title="Notes">
+        <MenuBar.Block count={80} label="All" />
+        <MenuBar.Block count={80} label="Users" />
+        <MenuBar.Block count={60} label="Leads" />
+        <MenuBar.Block count={60} label="Visitors" />
+        <MenuBar.SubTitle
+          iconProps={[
+            {
+              icon: Search,
+              onClick: () => setIsSearchCollapsed(!isSearchCollapsed),
```

Comment:
https://academy.bigbinary.com/learn-react/pages/know-when-to-use-previous-state-for-state-update